TAS_Engine/Convert/Environment_oM/InternalGain.cs

```diff
@@ -40,60 +41,89 @@ public static partial class Convert
         [Description("BH.Engine.TAS.Convert ToBHoM => gets BH.oM.Environment.Elements.InternalGain from TAS TBD InternalGain")]
         [Input("tbdInternalGain", "TAS TBD InternalGain")]
         [Output("BHoM Environmental InternalGain object")]
-        public static BHE.InternalGain ToBHoM(this TBD.InternalGain tbdInternalGain)
+        public static List<BHE.Gain> ToBHoM(this TBD.InternalGain tbdInternalGain)
         {
             if (tbdInternalGain == null) return null;
 
-            BHE.InternalGain internalGain = new BHE.InternalGain();
-            internalGain.Name = tbdInternalGain.name;
-            internalGain.Illuminance = tbdInternalGain.targetIlluminance;
-            internalGain.OutsideAirRatePerPerson = tbdInternalGain.freshAirRate;
-            internalGain.PersonGain = tbdInternalGain.personGain;
-            internalGain.RadiationProperties.EquipmentRadiation = tbdInternalGain.equipmentRadProp;
-            internalGain.RadiationProperties.LightingRadiation = tbdInternalGain.lightingRadProp;
-            internalGain.RadiationProperties.OccupantRadiation = tbdInternalGain.occupantRadProp;
-            internalGain.CoefficientProperties.EquipmentViewCoefficient = tbdInternalGain.equipmentViewCoefficient;
-            internalGain.CoefficientProperties.LightingViewCoefficient = tbdInternalGain.lightingViewCoefficient;
-            internalGain.CoefficientProperties.OccupantViewCoefficient = tbdInternalGain.occupantViewCoefficient;
-
-            internalGain.Profiles = tbdInternalGain.Profiles();
+            List<BHE.Gain> gains = new List<BHE.Gain>();
 
             Dictionary<string, object> tasData = new Dictionary<string, object>();
             tasData.Add("InternalGainActivityID", tbdInternalGain.activityID);
             tasData.Add("InternalGainDescription", tbdInternalGain.description);
             tasData.Add("InternalDomesticHotWater", tbdInternalGain.domesticHotWater);
 
-            internalGain.CustomData = tasData;
-            return internalGain;
+            BHE.Gain occupantGain = new BHE.Gain();
+            occupantGain.Name = tbdInternalGain.name;
+
+            BHP.GainPropertiesPeople peopleGain = new BHP.GainPropertiesPeople();
+            peopleGain.GainType = BHE.GainType.People;
+            peopleGain.GainUnit = BHE.GainUnit.NumberOfPeople;
+            peopleGain.RadiantFraction = tbdInternalGain.occupantRadProp;
+            peopleGain.ViewCoefficient = tbdInternalGain.occupantViewCoefficient;
+            peopleGain.SensibleGain = tbdInternalGain.personGain; //ToDo - review if this is the best place for this!
```

Comment:
this will be more tricky tbdInternalGain.personGain

_Metabolic Rate  - The number of Watts per person transferred to the zone as heat. 
The occupancy gains (sensible and latent) in W/m^2 divided by the metabolic rate (W/p) will give the specified people density for the zone in p/m^2._

![image](https://user-images.githubusercontent.com/26113670/52637813-45268500-2ec8-11e9-9ec1-9447f487f341.png)